sio-digital-team/active-safe
# Advanced custom fields

https://trello.com/c/zcLblAaU

**Changes:**
- [x] Added ACF to theme folder that auto initiates
- [x] Added settings to auto-generate and load settings with sync option for ACF
- [x] Added `Sports Landing Page` template to use the custom fields for Overview
- [x] Replace dummy text with content coming in from ACF

**How to Test:**
- Load the theme
- Go into `Custom Fields` and an option called `sync available()` should be seen. Use this to sync all fields.
- Create a new page and set the template to `Sports Landing Page`
- Add some text in the field and safe
- View the page and see if the text appears
